### ISSUE TITLE
Chronic.parse('09/01/00') returns nil

### ISSUE THREAD

**Issue by kobaltz** [mojombo/chronic#358]
```
pry(main)> Chronic.parse('09/01/99')
=> 1999-09-01 12:00:00 -0400
pry(main)> Chronic.parse('09/01/00')
=> nil
```
I would expect similar behaviour for parsing `YY` when it is `99` or `00`. I did not think that the y2k bug would still be haunting me 18 years later.

**Comment by davispuh** [comment]
Fixed with #359

**Comment by lawnknee** [comment]
Hey there, recently ran into this same exact issue despite there being a fix for it. Should I open up a new bug ticket?

<img width="405" alt="Screenshot 2024-08-09 at 7 44 32 PM" src="https://github.com/user-attachments/assets/023d68d4-801f-44c1-adce-ea7c29253e40">

Update -- just found this issue: https://github.com/mojombo/chronic/issues/387 so I might have misunderstood. It appears that the issue is fixed, but just a matter of releasing the code? 🙏
